cli/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ _Released 4/11/2024_
 - Suppresses benign warnings that reference Vulkan on GPU-less hosts. Addresses [#29085](https://github.com/cypress-io/cypress/issues/29085). Addressed in [#29278](https://github.com/cypress-io/cypress/pull/29278).
 
 
+**Bugfixes:**
+
+ - Specs with () in the filename will no longer fail to load and now behave as any other spec. Fixes [#28509](https://github.com/cypress-io/cypress/issues/28509)
```

Comment:
```suggestion
 - Specs with () in the filename will no longer fail to load and now behave as any other spec when run with the `--spec` argument via `cypress run`. Fixes [#28509](https://github.com/cypress-io/cypress/issues/28509)
```

Reply:
We also need to move this to a new change entry since 13.7.3 was released yesterday